docs/source/10_deployment/12_dask.md

```diff
@@ -0,0 +1,311 @@
+            for data_set in node.inputs:
+                load_counts[data_set] -= 1
+                if load_counts[data_set] < 1 and data_set not in pipeline.inputs():
+                    catalog.release(data_set)
+            for data_set in node.outputs:
+                if load_counts[data_set] < 1 and data_set not in pipeline.outputs():
+                    catalog.release(data_set)
+
+    def run_only_missing(
+        self, pipeline: Pipeline, catalog: DataCatalog
+    ) -> Dict[str, Any]:
+        """Run only the missing outputs from the ``Pipeline`` using the
+        datasets provided by ``catalog``, and save results back to the
+        same objects.
+
+        Args:
+            pipeline: The ``Pipeline`` to run.
+            catalog: The ``DataCatalog`` from which to fetch data.
+        Raises:
+            ValueError: Raised when ``Pipeline`` inputs cannot be
+                satisfied.
+
+        Returns:
+            Any node outputs that cannot be processed by the
+            ``DataCatalog``. These are returned in a dictionary, where
+            the keys are defined by the node outputs.
+        """
+        free_outputs = pipeline.outputs() - set(catalog.list())
+        missing = {ds for ds in catalog.list() if not catalog.exists(ds)}
+        to_build = free_outputs | missing
+        to_rerun = pipeline.only_nodes_with_outputs(*to_build) + pipeline.from_inputs(
+            *to_build
+        )
+
+        # We also need any missing datasets that are required to run the
+        # `to_rerun` pipeline, including any chains of missing datasets.
+        unregistered_ds = pipeline.data_sets() - set(catalog.list())
+        # Some of the unregistered datasets could have been published to
+        # the scheduler in a previous run, so we need not recreate them.
+        missing_unregistered_ds = {
+            ds_name
+            for ds_name in unregistered_ds
+            if not self.create_default_data_set(ds_name).exists()
+        }
+        output_to_unregistered = pipeline.only_nodes_with_outputs(
+            *missing_unregistered_ds
+        )
+        input_from_unregistered = to_rerun.inputs() & missing_unregistered_ds
+        to_rerun += output_to_unregistered.to_outputs(*input_from_unregistered)
+
+        # We need to add any previously-published, unregistered datasets
+        # to the catalog passed to the `run` method, so that it does not
+        # think that the `to_rerun` pipeline's inputs are not satisfied.
+        catalog = catalog.shallow_copy()
+        for ds_name in unregistered_ds - missing_unregistered_ds:
+            catalog.add(ds_name, self.create_default_data_set(ds_name))
+
+        return self.run(to_rerun, catalog)
+```
+
+### Update CLI implementation
+
+You're nearly there! Before being able to use the new runner, update the `run()` function in your `cli.py` file to make sure the runner class is instantiated correctly:
+
+```python
+def run(tag, env, parallel, ...):
```

Comment:
I understand that this saves space, but I feel like it would be very confusing to somebody less familiar with Kedro to figure out what to do here. This is especially true given `cli.py` has not been included as part of the project template since 0.17.5, and I don't see step-by-step documentation on how to set something like this up (e.g. I know I can go back to 0.17.4 template to find something like what I need).